.github/workflows/checks.yml

```diff
@@ -1,3 +1,4 @@
+name: Django-import-export-extensions CI
```

Comment:
Let's find better name, actually this actions perform checks, and `CI` is too common name